tests/subsys/cpp/libcxx/testcase.yaml

```diff
@@ -9,7 +9,6 @@ tests:
     min_flash: 54
     tags: cpp
   cpp.libcxx.exceptions:
-    arch_exclude: nios2
```

Comment:
we probably need to platform_exclude: qemu_nios2 here.

Reply:
For the time being, `twister` automatically skips any tests with `FLASH overflow` and treats it as a non-failure. Also, If we ever manage to slim down Zephyr such that it can fit into the current 128KB flash, it will pass anyway.
```
stephanos@devworx1:~/Dev/zephyrproject/zephyr$ scripts/twister -v -N -s tests/subsys/cpp/libcxx/cpp.libcxx.exceptions -p qemu_nios2
ZEPHYR_BASE unset, using "/home/stephanos/Dev/zephyrproject/zephyr"
Renaming output directory to /home/stephanos/Dev/zephyrproject/zephyr/twister-out.14
INFO    - Zephyr version: v2.6.0-rc2-57-g053ff6fa0e72
INFO    - JOBS: 60
INFO    - Using 'zephyr' toolchain.
INFO    - Building initial testcase list...
INFO    - 1 test scenarios (1 configurations) selected, 0 configurations discarded due to filters.
INFO    - Adding tasks to the queue...
INFO    - Added initial list of jobs to queue
INFO    - 1/1 qemu_nios2                tests/subsys/cpp/libcxx/cpp.libcxx.exceptions      SKIPPED (FLASH overflow)

INFO    - 0 of 0 test configurations passed (0.00%), 0 failed, 1 skipped with 0 warnings in 11.06 seconds
INFO    - In total 0 test cases were executed, 4 skipped on 1 out of total 372 platforms (0.27%)
INFO    - 0 test configurations executed on platforms, 0 test configurations were only built.
INFO    - Saving reports...
INFO    - Writing xunit report /home/stephanos/Dev/zephyrproject/zephyr/twister-out/twister.xml...
INFO    - Writing xunit report /home/stephanos/Dev/zephyrproject/zephyr/twister-out/twister_report.xml...
INFO    - Run completed
```

In that sense, I doubt it is necessary to `platform_exclude: qemu_nios2` here.

@nashif any comments on this?

Reply:
Ok, if that's the case than no issues.